examples/one.sh

```diff
@@ -13,10 +13,21 @@ has_target() {
   [ $RESULT_T -eq 0 ]
 }
 
+mo_examples() {
+  for i in ${1}; do
+    mo -s=../../DEPS "${i}" > "${i}.mo" ; mv "${i}.mo" "${i}";
+    echo "file ${i} executed against mo"
+  done
+}
+
 echo "$MODE $e"
 echo
 (
   cd "$e" || exit 1
+  dockerfiles=$(find . -iname Dockerfile)
+  if [ -n "${dockerfiles}" ]; then
+    mo_examples "${dockerfiles}"
```

Comment:
I understand now why you did it this way... I think I still would prefer to not use mo, but you can push me a little bit if you think I'm wrong.